### ISSUE TITLE
Armor Duping Glitch

### ISSUE THREAD

**Issue by UmbraBaruuk** [Iskallia/Vault-public#94]
Found a Glitch where by right clicking armor in your hot bar while in the crafting menu it would duplicate or occasionally triplicate armor pieces. Did some testing and found it worked on all armor regardless of armor typing and whether it was vanilla or modded (including vault armor). also found that by using hotbar spaces 1-5 you could force and duplicate the armor into specific slots; 1 - feet, 2 - legs, 3 - chest, 4 - head, and 5 - off hand

**Comment by magicalex234** [comment]
Had this same issue. However, i believe it's an issue with quark, and not vault hunters (but i could be wrong)

**Comment by ImreBoersma** [comment]
Duplicate issue with #62 #49 and #27, #72, #77

**Comment by Taticles** [comment]
I had an almost similar glitch with iron golum head which is kind of armour as can wear on head so thought it best be put here. I put a video of me doing it [here](https://youtu.be/jiRU5nWMyzs).

**Comment by magicalex234** [comment]
Yes, it works with any item you can equip to one of your armor slots
